Makefile

```diff
@@ -145,7 +145,7 @@ test:
 	cargo test --workspace --locked --features runtime-benchmarks,all-frequency-features
 
 integration-test:
-	./scripts/run_integration_tests.sh
+	./scripts/run_integration_tests.sh $(ARGS)
```

Comment:
This works, but forces the user to really read the docs in order to understand how to run tests. If we create multiple targets in the makefile, we can take advantage of makefile CLI auto-complete in some shells so that the user can hit TAB to see the available commands. Targets might look like this:

```
integration-test:
   ./scripts/run_integration_tests.sh
   
integration-test-only:
   ./scripts/run_integration_tests.sh -s
   
integration-test-load:
   ./scripts/run_integration_tests.sh load
   
integration-test-load-only:
   ./scripts/run_integration_tests.sh -s load
```

Then, if the user types `make int<TAB>`, they'll get:
```
integration-test integration-test-only integration-test-load integration-test-load-only
```

Reply:
Good idea, refactoring it in.